service_directory/tests/test_api.py

```diff
@@ -965,6 +965,17 @@ def setUpTestData(cls):
         )
         kwc.full_clean()  # force model validation to happen
 
+    def test_get_with_show_in_home_param(self):
+        Keyword.objects.create(
+            name='test', show_on_home_page=True
+        )
+        response = self.client.get(
+            '/api/keywords/', data={'show_on_home_page': True},
+            format='json'
+        )
+
+        self.assertEqual(1, len(response.data))
```

Comment:
this test checks if the response data

Reply:
no not that I can think of, the only other way I can think of is `assert response.data` but then that won't tell me much since I want to confirm that only 1 keyword is returned as there is only one that has the flag `show_on_home_page`